docs/docs/deploy/baremetal.md

```diff
@@ -24,9 +30,12 @@ The baremetal deploy runs several commands in sequence. These can be customized,
 3. `yarn rw prisma generate` - generates latest Prisma Client libs
 4. `yarn rw dataMigrate up` - runs data migrations, ignoring them if not installed
 5. `yarn rw build` - builds the web and/or api sides
-6. `yarn pm2 restart [service]` - restarts the serving process(es)
+6. `yarn pm2 restart [service]` - restarts the serving process(es).
 
-There is a special `--first-run` flag which can be included in your deploy command the first time you run it, which starts the PM2 services rather than restarting them.
+### First run lifecycle
+If the `--first-run` flag is specified step 6. aboce will be skipped and the following commands will run instead:
+  - `yarn pm2 start [service]` - starts the serving process(es)
+  - `yarn pm2 save` - saves the running services to the deploy users config file for future startup. See `First Deploy - Starting on Reboot` for further information
```

Comment:
Instead of making `First Deploy - Starting on Reboot` a code block, you can just leave it plain text but make it a link to the section below